MapboxNavigation/JunctionView.swift

```diff
@@ -50,18 +51,30 @@ public class JunctionView: UIImageView {
                 imageRepository.imageWithURL(guidanceViewImageURL, cacheKey: guidanceView.cacheKey!) { [unowned self] (downloadedImage) in
                     
                     self.isCurrentlyVisible = true
-                    self.isHidden = !self.isCurrentlyVisible
                     
                     DispatchQueue.main.async {
+                        self.isHidden = !self.isCurrentlyVisible
                         self.image = downloadedImage
-                        self.show(true)
                     }
                 }
             }
         }
         
     }
     
+    public func updateDistance(for currentStepProgress: RouteStepProgress) {
+        let distanceTravelled = currentStepProgress.distanceTraveled
```

Comment:
Nit: use the American spelling, `distanceTraveled`, for consistency with `RouteStepProgress.distanceTraveled`.

Reply:
Jolly good.